src/color/creating_reading.js

```diff
@@ -532,13 +545,13 @@ p5.prototype.lightness = function(c) {
  * </code>
  * </div>
  *
- * <div>
+ * <div class="norender">
  * <code>
- * colorMode(RGB, 255);
+ * colorMode(RGB, 255); // Set R in RGB range to 255
```

Comment:
Technically not setting only R in RGB to range but all values in RGB, only when providing 4 parameters do they indicate individual colour's range

Reply:
I'm not sure because I didn't write this example, I added comments to try and make sense of it but really I think it should be removed as it's so confusing.

Reply:
@hellonearthis how about this? I think this example is worth keeping since setting R, G, and B to the same range is a pretty common use case.
```suggestion
 * colorMode(RGB, 255); // Sets the range for red, green, and blue to 255
```